mobile/lib/features/wallet/create_invoice_screen.dart

```diff
@@ -222,3 +230,44 @@ class _CreateInvoiceScreenState extends State<CreateInvoiceScreen> {
     );
   }
 }
+
+void showNoLiquidityDialog() {
+  showDialog(
+      context: rootNavigatorKey.currentContext!,
+      builder: (context) => AlertDialog(
+              title: const Text("No liquidity in the LSP."),
+              content: RichText(
+                text: TextSpan(
+                  style: DefaultTextStyle.of(context).style,
+                  children: <TextSpan>[
+                    const TextSpan(
+                        text: "Not enough liquidity in the LSP to open channel.\n"
+                            "Please try again later.\n If you have further questions, please contact us on Telegram: ",
```

Comment:
This dialog needs a bit more love, but this space here adds a space at the beginning of the next line.

```suggestion
                            "Please try again later.\nIf you have further questions, please contact us on Telegram: ",
```